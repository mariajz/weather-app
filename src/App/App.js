import React from 'react';
import AppWrapper from '../components/app-wrapper';
import NavigationService from '../navigations';
import GlobalEvents from '../components/global-events';

const App = () => {
    return (
        <AppWrapper>
            <GlobalEvents />
            <NavigationService />
        </AppWrapper>
    );
};

export default App;
