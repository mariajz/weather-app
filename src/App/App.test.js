import React from 'react';
import { render } from '@testing-library/react-native';
import App from './App';

describe('App', () => {
    it('should contain the heading', () => {
        const container = render(<App />);
        const { getByTestId } = container;

        expect(getByTestId('heading').props.children).toStrictEqual(
            'Hello World!',
        );
        expect(container).toMatchSnapshot();
    });
});
