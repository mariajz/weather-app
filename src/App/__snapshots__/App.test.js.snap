// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render the app 1`] = `
<View>
  <View>
    Global Events
  </View>
  <View>
     Navigation service
  </View>
</View>
`;
